.github/workflows/dev-build.yml

```diff
@@ -19,8 +19,7 @@ jobs:
           trigger: '#deploy-me'
           reaction: rocket
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_CI_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is this `GITHUB_TOKEN`?

Reply:
well i did a mistake in my previous pr about this specific step + semantic release

There are a big difference between `GITHUB_TOKEN` and `GH_ACCESS_TOKEN`.
`GITHUB_TOKEN` is for an internal secret that github provides to ci (Automatic token authentication)
`GH_ACCESS_TOKEN` is the one we needed to replace (the one has correct access to qlik-trial)

So i confused those two in my prev pr (so sorry about it)